tests/Functional/Command/MauticLanguagePackerCommandTest.php

```diff
@@ -0,0 +1,608 @@
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpFoundation\Request;
+
+class MauticLanguagePackerCommandTest extends KernelTestCase
+{
+    use ResponseBodyBuilderTrait;
+
+    private CommandTester $commandTester;
+
+    private Filesystem $filesystem;
+
+    private string $packagesDir;
+
+    private string $translationsDir;
+
+    private MockHandler $mockHandler;
+
+    private AwsMockHandler $awsMockHandler;
+
+    protected function setUp(): void
+    {
+        $container        = self::getContainer();
+        $parameterBag     = $container->get('parameter_bag');
+        $this->filesystem = $container->get('filesystem');
+
+        $application = new Application();
+        $application->add(
+            new MauticLanguagePackerCommand(
+                $parameterBag,
+                $this->filesystem,
+                $container->get(ResourcesService::class),
+                $container->get(BuildPackageService::class),
+                $container->get(UploadPackageService::class)
+            )
+        );
+        $command             = $application->find(MauticLanguagePackerCommand::NAME);
+        $this->commandTester = new CommandTester($command);
+
+        $this->packagesDir     = $parameterBag->get('mlp.packages.dir');
+        $this->translationsDir = $parameterBag->get('mlp.translations.dir');
+
+        $this->mockHandler    = $container->get('http.client.mock_handler');
+        $this->awsMockHandler = $container->get('aws.client.mock_handler');
+    }
+
+    /**
+     * @dataProvider provideExecutionData
+     *
+     * @param ResponseInterface[]                   $mockResponses
+     * @param array<string, string[]>               $commandArguments
+     * @param ResultInterface[]|\RuntimeException[] $awsMockResponses
+     */
+    public function testExecute(
+        string $expectedOutput,
+        array $mockResponses,
+        array $commandArguments = [],
+        array $awsMockResponses = []
+    ): void {
+        foreach ($mockResponses as $mockResponse) {
+            $this->mockHandler->append($mockResponse);
+        }
+
+        foreach ($awsMockResponses as $awsMockResponse) {
+            $this->awsMockHandler->append($awsMockResponse);
+        }
+
+        $this->commandTester->execute($commandArguments);
+        Assert::assertStringContainsString($expectedOutput, $this->getFixedCommandOutput());
+    }
+
+    /**
+     * @return array<mixed>
+     */
+    public static function provideExecutionData(): iterable
+    {
+        $container       = self::getContainer();
+        $parameterBag    = $container->get('parameter_bag');
+        $translationsDir = $parameterBag->get('mlp.translations.dir');
+
+        $organisation = $_ENV['TRANSIFEX_ORGANISATION'];
+        $project      = $_ENV['TRANSIFEX_PROJECT'];
```

Comment:
```suggestion
        $transifexConfig = $container->get(\Mautic\Transifex\ConfigInterface::class);

        $organisation = $transifexConfig->getOrganization();
        $project      = $transifexConfig->getProject();
```
Let's avoid the global state.